contrib/systemd/containerized-installs/openvswitch.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+After=docker.service
+Requires=docker.service
+Before=origin-master.service
+
+[Service]
+EnvironmentFile=/etc/sysconfig/origin-node
+ExecStartPre=-/usr/bin/docker rm -f openvswitch
```

Comment:
Clean up a bad exit.  If docker client process gets SIGKILL the container
will still exist.

On Wed, Nov 4, 2015 at 3:36 PM, Eric Paris <notifications@github.com> wrote:

> In contrib/systemd/containerized-installs/openvswitch.service
> <https://github.com/openshift/origin/pull/5442#discussion_r43935760>:
>
> > @@ -0,0 +1,14 @@
> > +[Unit]
> > +After=docker.service
> > +Requires=docker.service
> > +Before=origin-master.service
> > +
> > +[Service]
> > +EnvironmentFile=/etc/sysconfig/origin-node
> > +ExecStartPre=-/usr/bin/docker rm -f openvswitch
>
> Why? You already have --rm?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/5442/files#r43935760>.
>